### PR TITLE
doc: Doc updates for 2.1.3 release

### PR DESCRIPTION
**Pull request by peknis**
Doc updates for NCS 2.1.3 bug fix release

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-2.1.3.rst:92]: We'll have to clean the changelog of this entry on the 2.1 branch, but we can wait until a day before release with that.

Comment (peknis) [reply doc/nrf/releases/release-notes-2.1.3.rst:92]: 2.1 branch?

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (peknis) [reply doc/nrf/releases/release-notes-2.1.3.rst:22]: Roger.

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (peknis) [reply doc/nrf/releases/release-notes-2.1.3.rst]: You missed my mistake in the last item (213 -> 212) ;)

**Review by nordicjm — CHANGES_REQUESTED** [review]
To be added to DFU and FOTA:

```
.. rst-class:: v2-1-2 v2-1-1 v2-1-0

NCSDK-18357: Serial recovery does not work on nRF5340 network core

  If a network core serial recovery is attempted using MCUboot serial recovery, the upload will complete, but the image will not be transferred to the network core and the upgrade will fail.

.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0

NCSDK-18422: Serial recovery fails to write to slots in QSPI

  If a slot resides in an external QSPI storage area and this area is written to in MCUboot's serial recovery system, the writing will not work due to memory buffer alignment offsets and requirements with the underlying flash driver.

.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0

NCSDK-18108: ``s1`` variant image configuration mismatch
  If an image with an ``s1`` variant is configured and the ``s0`` image configuration is changed using menuconfig, these changes will not be reflected in the ``s1`` configuration, which can lead to a differing build configuration or the build does not upgrade.
```

**Comment by peknis** [comment]
Are these issues still present in 2.1.3?

**Comment by nordicjm** [comment]
> Are these issues still present in 2.1.3?

No, PRs to fix them have been submitted (not yet approved) here:
https://github.com/nrfconnect/sdk-nrf/pull/9600
https://github.com/nrfconnect/sdk-nrf/pull/9601

**Comment by nordicjm** [comment]
@peknis it seems like these issues are still present in 2.1.3 as the fixes are not being accepted, so will need 2.1.3 added.

**Comment by peknis** [comment]
> @peknis it seems like these issues are still present in 2.1.3 as the fixes are not being accepted, so will need 2.1.3 added.

Done

**Review by martintv — APPROVED** [review]

**Review by czeslawmakarski — APPROVED** [review]

**Review by nordicjm — APPROVED** [review]

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (rlubos) [reply doc/nrf/releases/release-notes-2.1.3.rst:129]: The command is correct.

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-2.1.3.rst:28]:
```suggestion
  * An issue in MPSL where the controller would assert when a Bluetooth role was running.
    This was previously listed as the known issue with the issue ID DRGN-17851.
  * An issue in MPSL where the controller would abandon a link, causing a disconnect on the remote side.
    This was previously listed as the known issue with the issue ID DRGN-18105.
  * An issue where RPA (Random Private Address) would not be changed when it should when BT_EXT_ADV was enabled.
```

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-2.1.3.rst:94]: Empty section. We should duplicate entries from highlights, because highlights just summarize the salient entries from the changelog. Ideally, the highlights entries should be shorter than the changelog ones, so you can for example skip mentioning the line with the ID in the highlights and mention it only in the changelog.

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-2.1.3.rst:117]: Empty sections. Either update or remove.

**Review by greg-fer — CHANGES_REQUESTED** [review]

**Review by LuDuda — APPROVED** [review]

**Comment by MarekPieta** [comment]
Hi @peknis 
Was the fix for nRF Desktop profiler sync module build failures (`NCSDK-18552`, commit hash on main: `28ff23ac26c079eb966893e9a64a624bf4f50b71`) included in this release? If so, then we may may want to note it here.

**Comment by peknis** [comment]
> Hi @peknis @nordicjm Was the fix for nRF Desktop profiler sync module build failures (`NCSDK-18552`, commit hash on main: `28ff23ac26c079eb966893e9a64a624bf4f50b71`) included in this release? If so, then we may may want to note it here.

I don't think that fix went into 2.1.3 release. Let's see what @rlubos says about this.

**Comment by MarekPieta** [comment]
@nordicjm informed me that 2.1.3 was meant to only fix 1 or 2 specific issues. In that case we may just need to update known-issue introduced here: https://github.com/nrfconnect/sdk-nrf/pull/9647 to mark that it replicates also in 2.1.3. Sorry for confusion.

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-2.1.3.rst:26]:
```suggestion
  * An issue where RPA (Random Private Address) would not be changed when it should when :kconfig:option:`CONFIG_BT_EXT_ADV` was enabled.
```
Isn't this a Kconfig option?

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (greg-fer) [inline doc/nrf/releases/release-notes-2.1.3.rst:113]: Just remove these sections if no changes were made to these components.

**Comment by rlubos** [comment]
> @nordicjm informed me that 2.1.3 was meant to only fix 1 or 2 specific issues. In that case we may just need to update known-issue introduced here: #9647 to mark that it replicates also in 2.1.3. Sorry for confusion.

That's true, we're only fixing BLE & Matter in this release. 

Regarding the release notes, do they need to be updated here, isn't it enough to update the relase notes page on `main`? I'd rather conclude this PR already given that we shoudl tag tomorrow.

**Review by greg-fer — APPROVED** [review]
Pushed a minor edit with a fix to line 98's Kconfig option. This was a push with force, @peknis , so please rebase before you push any potential edit.

**Review by greg-fer — APPROVED** [review]

File: doc/nrf/known_issues.rst
Comment (tejlmand) [inline doc/nrf/known_issues.rst:1556]: Somewhere in this section add the following known issue:
```suggestion
NCSIDB-816: nanopb / protoc not specified correctly by the nRF Connect SDK Toolchain.
  nRF Connect SDK Toolchain includes nanopb / protoc when installed but the path to the tool is not propagated correctly to the nRF Connect SDK build system.
  
  ** Workaround** Find the location of the protoc executable in the nRF Connect SDK and manually its path your system's global path
```

**Review by tejlmand — COMMENTED** [review]
text proposal for nanopb issue.

File: doc/nrf/known_issues.rst
Comment (peknis) [reply doc/nrf/known_issues.rst:1556]: Thanks. Is this fixed in NCS 2.2.0? If not, I'll also add the tag for it in the PR I'll open to update main.

File: doc/nrf/known_issues.rst
Comment (greg-fer) [inline doc/nrf/known_issues.rst:1555]: Is the nanopb issue valid for all releases since 1.3.0? @peknis , @tejlmand ?

**Review by greg-fer — APPROVED** [review]
Approving, pending answer to one final comment.

File: doc/nrf/known_issues.rst
Comment (RaoulSP) [inline doc/nrf/known_issues.rst:1560]:
```suggestion
  ** Workaround:** Locate the nanopb ``generator-bin`` directory in your |NCS| Toolchain, and add its path to your system's global path. This allows the protoc executable to be found.
```

File: doc/nrf/known_issues.rst
Comment (tejlmand) [reply doc/nrf/known_issues.rst:1556]: No, it's not fixed in v2.2.0

File: doc/nrf/known_issues.rst
Comment (tejlmand) [reply doc/nrf/known_issues.rst:1555]: no, it's only a valid issue from v2.0.0, cause before that nanopb - protoc was not part of the toolchain and uesrs at that time were requested to install this tool manually.

Comment (tejlmand) [reply doc/nrf/known_issues.rst:1555]:
```suggestion
.. rst-class:: v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

**Review by tejlmand — DISMISSED** [review]

**Comment by PavelVPV** [comment]
Please add the following:
```
UART driver fixes:

- Fixed an issue where re-enabling UART after uart_rx_disable() too early could cause random hardware faults (NCSIDB-875)
- Fixed an issue where NULL pointer de-referencing happens if PM_DEVICE_ACTION_RESUME action was requested (NCSDK-16753)
```

**Comment by greg-fer** [comment]
> Please add the following:

@PavelVPV , was this implemented in the nrfx module or elsewhere? Looking at the PR @rlubos mentioned, I think these lines should go to Zephyr section of the docs?

**Comment by PavelVPV** [comment]
> > Please add the following:
> 
> @PavelVPV , was this implemented in the nrfx module or elsewhere? Looking at the PR @rlubos mentioned, I think these lines should go to Zephyr section of the docs?

It is implemented in Zephyr, not in nrfx module.

File: doc/nrf/known_issues.rst
Comment (umapraseeda) [inline doc/nrf/known_issues.rst:14]: Shouldn't it be `known issues for nRF Connect SDK v2.1.3`_  rather than :ref:`known_issues`?
```suggestion
   You are looking at the page for the patch release. For the latest version of the page that tracks updates on the ``main`` branch, see :ref:`known_issues`.
```

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.1.3.rst:30]:
```suggestion
  * Fixed an issue where re-enabling UART after calling :c:func:`uart_rx_disable` too early could cause random hardware faults.
```

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.1.3.rst:100]:
```suggestion
* An issue in MPSL where the controller would assert when a Bluetooth role was running (DRGN-17851).
```

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.1.3.rst:102]:
```suggestion
* An issue in MPSL where the controller would abandon a link, causing a disconnect on the remote side (DRGN-18105).
```

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.1.3.rst:103]:
```suggestion
* An issue where RPA (Random Private Address) would not be changed when it should when ``BT_EXT_ADV`` was enabled.
```

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.1.3.rst:115]:
```suggestion
  * Fixed an issue where re-enabling UART after calling :c:func:`uart_rx_disable` too early could cause random hardware faults.
```

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.1.3.rst:116]:
```suggestion
  * Fixed an issue where NULL pointer was de-referenced if ``PM_DEVICE_ACTION_RESUME`` action was requested.
```

File: doc/nrf/releases/release-notes-changelog.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-changelog.rst:14]: Shouldn't this be https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases/release-notes-changelog.html rather than :ref:`release_notes`?
```suggestion
   You are looking at the page for the patch release. For the latest version of the page that tracks updates on the ``main`` branch, see :ref:`release_notes`.
```

File: doc/nrf/known_issues.rst
Comment (peknis) [reply doc/nrf/known_issues.rst:14]: Not sure about this.

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.1.3.rst:97]:
```suggestion
Fixed:
```

File: doc/nrf/releases/release-notes-2.1.3.rst
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-2.1.3.rst:116]: Should we modify as follows:
```suggestion
  * Fixed:
  
    * An issue where re-enabling UART after calling :c:func:`uart_rx_disable` too early could cause random hardware faults.
    * An issue where NULL pointer was de-referenced if ``PM_DEVICE_ACTION_RESUME`` action was requested.
```

**Comment by rlubos** [comment]
@tejlmand Please approve

**Review by umapraseeda — APPROVED** [review]

**Pull request merged**
